functions/exercise2/index.html

```diff
@@ -5,14 +5,20 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>excercise2</title>
+    <link rel="stylesheet" href="style.css" />
+    <title>excercise10</title>
     <script src="script.js"></script>
 </head>
 
 <body>
     <h1>Palindrome</h1>
-    Phrase:<input id="phrase" />
-    <button onclick="showResult()">Click here to check if your phrase is palindrome</button>
-    <output id="result" />
+    <div> Phrase:<input id="phrase" /></div>
```

Comment:
Do we have to surround it with div? Can't we use different way of positoning it?